dataplane/standalone/router_interface.cc

```diff
@@ -14,10 +14,41 @@
 
 #include "dataplane/standalone/router_interface.h"
 
+#include <glog/logging.h>
+
+#include <vector>
+
 #include "dataplane/standalone/translator.h"
 
 sai_status_t RouterInterface::create(_In_ uint32_t attr_count,
                                      _In_ const sai_attribute_t* attr_list) {
+  std::vector<sai_attribute_t> attrs(attr_list, attr_list + attr_count);
+  sai_router_interface_type_t type;
+  for (auto attr : attrs) {
+    switch (attr.id) {
+      // TODO(dgrau): Handle vr and mtu.
+      case SAI_ROUTER_INTERFACE_ATTR_VIRTUAL_ROUTER_ID:
+        break;
+      case SAI_ROUTER_INTERFACE_ATTR_MTU:
+        break;
+      case SAI_ROUTER_INTERFACE_ATTR_TYPE:
+        type = static_cast<sai_router_interface_type_t>(attr.value.s32);
+        break;
+    }
+  }
+  if (type == SAI_ROUTER_INTERFACE_TYPE_LOOPBACK) {
+    grpc::ClientContext context;
+    lemming::dataplane::CreatePortRequest req;
+    lemming::dataplane::CreatePortResponse resp;
+    req.set_id(this->id);
+    req.set_type(forwarding::PORT_TYPE_KERNEL);
```

Comment:
Is this the right type? How does a loopback interface work on lemming?

Reply:
for now using the kernel's loopback interface (lo)